db/models.py

```diff
@@ -50,3 +61,46 @@ class MovieSession(models.Model):
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(default=now)
```

Comment:
The `created_at` field should use `auto_now_add=True` to automatically set the field during creation, as specified in the task requirements. The current implementation using `default=now` does not align with this requirement.